layout/layout.h

```diff
@@ -171,7 +171,17 @@ enum {
 	M0_LAYOUT_ENUM_TYPE_MAX = 32
 };
 
-enum { M0_DEFAULT_LAYOUT_ID = 1 };
+#ifdef CONFIG_X86_64
+	enum { M0_DEFAULT_LAYOUT_ID = 1 };
+#elif defined CONFIG_AARCH64
+	enum { M0_DEFAULT_LAYOUT_ID = 5 }; 
```

Comment:
Drop trailing spaces here and in all other places in your patch, please.

Reply:
fixed

Reply:
Not everywhere:

```
$ git diff origin/main | grep -E '^\+.* +$'
+%configure %{with_linux} %{with_lustre} %{configure_opts} 
+%configure %{with_linux} %{with_lustre} %{configure_opts} enable_castxml=yes  
+}; 
+	enum { M0_DEFAULT_LAYOUT_ID = 5 }; 
+		  
+   Implementation of atomic operations for Linux user space uses aarch64 
+   assembly language instructions (with gcc syntax). The aarch64 uses its own set of atomic 
+   assembely instruction to ensure atomicity ---no optimisation for non-SMP 
+/** 
+   Implementation of atomic operations for Linux user space uses x86_64 
+   everywhere to ensure atomicity 
+    #define PAGE_SHIFT    16 
```

Reply:
See in which files:

```
$ git diff origin/main | grep -E '^diff|^\+.* +$' | grep -E -B1 '^\+'
diff --git a/cortx-motr.spec.in b/cortx-motr.spec.in
+%configure %{with_linux} %{with_lustre} %{configure_opts} 
+%configure %{with_linux} %{with_lustre} %{configure_opts} enable_castxml=yes  
--
diff --git a/dtm/remote.c b/dtm/remote.c
+}; 
--
diff --git a/layout/layout.h b/layout/layout.h
+	enum { M0_DEFAULT_LAYOUT_ID = 5 }; 
--
diff --git a/lib/Kbuild.sub b/lib/Kbuild.sub
+		  
--
diff --git a/lib/user_space/user_aarch64_atomic.h b/lib/user_space/user_aarch64_atomic.h
+   Implementation of atomic operations for Linux user space uses aarch64 
+   assembly language instructions (with gcc syntax). The aarch64 uses its own set of atomic 
+   assembely instruction to ensure atomicity ---no optimisation for non-SMP 
+/** 
--
diff --git a/lib/user_space/user_x86_64_atomic.h b/lib/user_space/user_x86_64_atomic.h
+   Implementation of atomic operations for Linux user space uses x86_64 
+   everywhere to ensure atomicity 
--
diff --git a/net/lnet/ut/lnet_ut.c b/net/lnet/ut/lnet_ut.c
+    #define PAGE_SHIFT    16 
```

Reply:
You can also use `git diff --check`.